src/error.rs

```diff
@@ -1,12 +1,11 @@
 use std::fmt;
 use std::result::Result as StdResult;
 
-use reqwest::Error as ReqwestError;
+use anyhow::Error as AnyhowError;
 use thiserror;
 
 /// A specialized `Result` type for rs-cnc.
-pub type Result<T> = StdResult<T, Error>;
-
+pub type Result<T> = StdResult<T, AnyhowError>;
```

Comment:
Oh nice! I've totally been making my life harder in astria-conductor. I see the value in Anyhow now.

Reply:
I love anyhow :D